### ISSUE TITLE
Receiving messages from Firebase while app is killed

### ISSUE THREAD

**Issue by MasterMind2k** [Larpon/QtFirebase#113]
During development I've identified that app, if killed, will not receive messages to C++ code if app is not running (with some moment to think about it, it makes sense actually :) ).

I have a use case that off the shelve notification bubble delivered by Firebase implementation won't do.

I am just opening this ticket to see if anyone has similar issues. I've cut some corners by completely ignoring Firebase C++ SDK and written a small layer in Java and custom integration API in the C++ parts.

Another thing that I realised is, that C++ written service cannot receive firebase messages. Since you need an Activity object when registering firebase app.

So, to sum this up. The use case is:
  * Be able to parse data messages even when application is dead

**Comment by Larpon** [comment]
Have you tried making a [service](https://doc.qt.io/qt-5/android-services.html)? That way you have C++ code still alive when the app is shut down?

**Comment by MasterMind2k** [comment]
Yes, I've made a use of services. But in this case, I do not need (or able) to use QtFirebase's wrapper around messaging.

Which in my use case, is not a problem. Maybe an idea, to prepare an infrastructure to have this ability by default in QtFirebase.

**Comment by Larpon** [comment]
@MasterMind2k - just to make sure; [are you using this](https://github.com/Larpon/QtFirebaseExample/blob/0adea209ee2558f1f2a7a4bc1ae18c0cb9253e7b/App/platforms/android/src/com/blackgrain/android/firebasetest/Main.java#L29-L77)

**Comment by MasterMind2k** [comment]
Hmm, not sure. I do have somewhere in my memory that I've tried it.

Will retry anyway and post results.
